packages/@aws-cdk/aws-ecs-patterns/lib/base/load-balanced-service-base.ts

```diff
@@ -198,6 +198,12 @@ export abstract class LoadBalancedServiceBase extends cdk.Construct {
     new cdk.CfnOutput(this, 'LoadBalancerDNS', { value: this.loadBalancer.loadBalancerDnsName });
   }
 
+  protected getCreateDefaultCluster(scope: cdk.Construct, vpc?: ec2.IVpc): ecs.Cluster {
+    const DEFAULT_CLUSTER_ID = `EcsDefaultCluster${vpc ? vpc.node.id : ''}`;
```

Comment:
"salt" the cluster ID with some uuid or something that will prevent people from accidentally defining constructs that will collide with this name, especially if it's `EcsDefaultCluster`.

Reply:
Do you mean a variable value or some magic string in the code to prevent collision?

Reply:
magic string